src/Features/LanguageServer/Protocol/Handler/OnAutoInsert/OnAutoInsertHandler.cs

```diff
@@ -41,7 +41,7 @@ internal class OnAutoInsertHandler : AbstractStatelessRequestHandler<LSP.Documen
             [ImportMany(LanguageNames.VisualBasic)] IEnumerable<IBraceCompletionService> visualBasicBraceCompletionServices)
         {
             _csharpBraceCompletionServices = csharpBraceCompletionServices.ToImmutableArray();
-            _visualBasicBraceCompletionServices = _visualBasicBraceCompletionServices.ToImmutableArray();
+            _visualBasicBraceCompletionServices = visualBasicBraceCompletionServices.ToImmutableArray();
```

Comment:
This looks like a bug!

@CyrusNajmabadi Any idea what this may have caused and how to test?

Reply:
Nope. @dibarbet ?

Reply:
I wonder if there was a test that could have shown an issue here.

Reply:
@CyrusNajmabadi I added a test. Haven't yet verified that it's failing before the change or passing after. Will test

Reply:
The test I added is unrelated to this line. It's not a brace completion service. Will see if I can figure out a test.

Reply:
I can't figure out a test. Most (if not all) of those VB services are unrelated to typing a new line, while `OnAutoInsertHandler` calls the services only if the character typed is a new line.